tests/test_e2e_15_mef_eline.py

```diff
@@ -424,3 +424,86 @@ def test_004_tag_restriction(self):
+        circuit_id = response.json()["circuit_id"]
+
+        expected = [[1, 9], [16, 3798], [3800, 4095]]
+        intf_id = '00:00:00:00:00:00:00:01:1'
+        api_url = KYTOS_API + f'/topology/v3/interfaces/{intf_id}/tag_ranges'
+        response = requests.get(api_url)
+        data = response.json()
+        assert response.status_code == 200, response.text
+        assert expected == data[intf_id]["available_tags"]["vlan"]
+
+        intf_id = '00:00:00:00:00:00:00:02:1'
+        api_url = KYTOS_API + f'/topology/v3/interfaces/{intf_id}/tag_ranges'
+        response = requests.get(api_url)
+        data = response.json()
+        assert response.status_code == 200, response.text
+        assert expected == data[intf_id]["available_tags"]["vlan"]
+
+        payload = {
+            "uni_a": {
+                "tag": {"tag_type": 1, "value": [[12, 21]]},
+                "interface_id": "00:00:00:00:00:00:00:01:1"
+            },
+            "uni_z": {
+                "tag": {"tag_type": 1, "value": [[13, 21]]},
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            }
+        }
+        api_url = KYTOS_API + f'/mef_eline/v2/evc/{circuit_id}'
+        response = requests.patch(api_url, json=payload)
+        assert response.status_code == 400, response.text
+
+        payload["uni_z"]["tag"]["value"] = [[12, 21]]
+        api_url = KYTOS_API + f'/mef_eline/v2/evc/{circuit_id}'
+        response = requests.patch(api_url, json=payload)
+        assert response.status_code == 200, response.text
+
+        expected = [[1, 11], [22, 3798], [3800, 4095]]
+        intf_id = '00:00:00:00:00:00:00:01:1'
+        api_url = KYTOS_API + f'/topology/v3/interfaces/{intf_id}/tag_ranges'
+        response = requests.get(api_url)
+        data = response.json()
+        assert response.status_code == 200, response.text
+        assert expected == data[intf_id]["available_tags"]["vlan"]
+
+        intf_id = '00:00:00:00:00:00:00:02:1'
+        api_url = KYTOS_API + f'/topology/v3/interfaces/{intf_id}/tag_ranges'
+        response = requests.get(api_url)
+        data = response.json()
+        assert response.status_code == 200, response.text
+        assert expected == data[intf_id]["available_tags"]["vlan"]
+
+        time.sleep(10)
+
+        s1, s2 = self.net.net.get('s1', 's2')
+        flows_s1 = s1.dpctl('dump-flows')
+        flows_s2 = s2.dpctl('dump-flows')
+        assert len(flows_s1.split('\r\n ')) == 8, flows_s1
+        assert 'in_port="s1-eth1",vlan_tci=0x100c/0x1ffc' in flows_s1
+        assert 'in_port="s1-eth1",vlan_tci=0x1010/0x1ffc' in flows_s1
+        assert 'in_port="s1-eth1",vlan_tci=0x1014/0x1ffe' in flows_s1
+        assert len(flows_s2.split('\r\n ')) == 8, flows_s2
+        assert 'in_port="s2-eth1",vlan_tci=0x100c/0x1ffc' in flows_s2
+        assert 'in_port="s2-eth1",vlan_tci=0x1010/0x1ffc' in flows_s2
+        assert 'in_port="s2-eth1",vlan_tci=0x1014/0x1ffe' in flows_s2
```

Comment:
Great flows assertion, for completeness, let's also ensure that ping is also working, ideally exercising these three flow entries.

Reply:
Added ping